pages/_app.js

```diff
@@ -1,32 +1,34 @@
-import Script from 'next/script'
-import '../styles/globals.css'
-import * as gtag from '../lib/gtag'
+import '../styles/globals.css';
+import * as gtag from '../lib/gtag';
+import Head from 'next/head';
+import { useEffect } from "react";
+import { useRouter } from 'next/router';
+import loadCookieControl from 'components/CookieControl';
 
 function MyApp({ Component, pageProps }) {
+  const router = useRouter();
+  const apiKey = process.env.NEXT_PUBLIC_API_KEY;
```

Comment:
Can you please rename this key to something more descriptive of what it's for? `NEXT_PUBLIC_API_KEY` is very vague - which API are we talking about? 😄 

Also, it would be probably be useful to add it to the `env.example` file.